pkgs/development/libraries/qt-5/modules/qtbase.nix

```
@@ -151,6 +151,9 @@ stdenv.mkDerivation {
     ''}
 
     NIX_CFLAGS_COMPILE+=" -DNIXPKGS_QT_PLUGIN_PREFIX=\"$qtPluginPrefix\""
+
+    # paralellize compilation of qtmake, which happens within ./configure
+    export MAKEFLAGS="-j$NIX_BUILD_CORES"
```

```suggestion
    export MAKEFLAGS+=" -j$NIX_BUILD_CORES"
```
Just noticed that appending is probably safer.